rocketpool-cli/node/sign-message.go

```diff
@@ -0,0 +1,57 @@
+package node
+
+import (
+	"fmt"
+
+	"github.com/urfave/cli"
+
+	"github.com/rocket-pool/smartnode/shared/services/rocketpool"
+	cliutils "github.com/rocket-pool/smartnode/shared/utils/cli"
+)
+
+func signMessage(c *cli.Context) error {
+
+	// Get RP client
+	rp, err := rocketpool.NewClientFromCtx(c)
+	if err != nil {
+		return err
+	}
+	defer rp.Close()
+
+	// Get & check wallet status
+	status, err := rp.WalletStatus()
+	if err != nil {
+		return err
+	}
+
+	if !status.WalletInitialized {
+		fmt.Println("The node wallet is not initialized.")
+		return nil
+	}
+
+	message := c.String("message")
+	if message == "" {
```

Comment:
```suggestion
	for message == "" {
```

This would ensure that `message != ""` (ie do minimal validation on stdin)

Reply:
Ok. But this validation was already done by the Prompt regex. (^.+$).

Reply:
Isn't `-m` optional?